src/Glpi/Form/Renderer/FormRenderer.php

```diff
@@ -85,7 +86,8 @@ public function render(Form $form): string
         // Load template
         $twig = TemplateRenderer::getInstance();
         $html .= $twig->render('pages/form_renderer.html.twig', [
-            'form' => $form,
+            'form'              => $form,
+            'is_anonymous_form' => FormAccessControlManager::getInstance()->isAnonymousForm($form),
```

Comment:
Small issue with that, a form might allow anonymous users but that doesn't mean that an authenticated user can't access it as well.
I think a form that can be anonymous shouldn't restrict questions for a legitimate non-anonymous user.

Maybe we should just check the session here ?